commune_app/models.py

```diff
@@ -1,3 +1,9 @@
-# from django.db import models
+from django.db import models
 
-# Create your models here.
+
+class Chore(models.Model):
+    # Commune = models.ForeignKey(Commune, on_delete=models.CASCADE)
+    title = models.CharField(max_length=50)
+    date = models.DateTimeField(auto_now_add=True)
+    assign = models.ForeignKey(User, on_delete=models.CASCADE)
+    description = models.TextField(max_length=2500)
```

Comment:
IMHO you can do it like:
```suggestion
    description = models.TextField(max_length=300)
```
2500 is too long for this purpose